homeassistant/components/tractive/__init__.py

```diff
@@ -210,6 +210,15 @@ async def _listen(self) -> None:
                     ):
                         self._last_pos_time = event["position"]["time"]
                         self._send_position_update(event)
+            except aiotractive.exceptions.UnauthorizedError:
+                _LOGGER.debug(
+                    "Tractive responds with unauthorized. Have you changed your password? Disabling integration"
+                )
+                async_dispatcher_send(
+                    self._hass, f"{SERVER_UNAVAILABLE}-{self._user_id}"
+                )
+                # How do I disable the integration till restart and show an error message in the HASS UI from here?
```

Comment:
We can use `persistent_notification` here to inform user about the problem.

Reply:
Or maybe `rapairs` platform should be use in such situation https://github.com/home-assistant/core/issues?q=label%3A%22integration%3A+repairs%22